cloudability/metrics-agent
# Adding the possibility to use a secret already present in Kubernetes

#### What does this PR do?
Makes the creation of the secret from the helm chart optional by allowing the user to specify `secretName`.

#### Where should the reviewer start?
Check the helm chart content

#### How should this be manually tested?
Deploy without specifying secretName to check retrocompatibility.
Upload the secret manually with a custom name and specify that name in `secretName`

#### Any background context you want to provide?
I have the necessity to manage the secrets outside the repo invoking the helm chart and not to have it in code.

#### What picture best describes this PR (optional but encouraged)?
Flexibility in the usage of secrets in order to increase security.

#### What are the relevant Github Issues?
Just a feature that I needed

#### Developer Done List


#### For the Reviewer:

#### By approving this PR, the reviewer acknowledges that they have checked all items in this done list.

#### Reviewer/Approval Done List
